.github/workflows/pr-checks.yml

```
@@ -158,3 +158,34 @@ jobs:
           # Test that the app can show SDK version
           ./build/taskscpp --ditto-sdk-version
           echo "✅ SDK version command works"
+
+  kotlin-multiplatform:
+    name: Kotlin Multiplatform (ubuntu-24.04)
+    runs-on: ubuntu-24.04
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up JDK 17
+        uses: actions/setup-java@v4
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@v4
+
+      - name: Create .env file
+        run: |
+          echo "DITTO_APP_ID=${{ secrets.DITTO_APP_ID }}" > .env
+          echo "DITTO_PLAYGROUND_TOKEN=${{ secrets.DITTO_PLAYGROUND_TOKEN }}" >> .env
+          echo "DITTO_AUTH_URL=${{ secrets.DITTO_AUTH_URL }}" >> .env
+          echo "DITTO_WEBSOCKET_URL=${{ secrets.DITTO_WEBSOCKET_URL }}" >> .env
+
+      - name: Build project
+        working-directory: kotlin-multiplatform
+        run: ./gradlew build
```

The standard gradle `build` task includes tests and lint checks. `assemble` is the build-only task.

```suggestion
        run: ./gradlew assemble
```